oni.sh

```diff
@@ -14,15 +14,15 @@ self=$0
 OPEN_DIRECTORY="$PWD"
 
 # test if $self is a symlink
-if [ -L $self ] ; then
+if [ -L "$self" ] ; then
```

Comment:
What is the purpose of these changes to `oni.sh`? Are they related to the menu changes, or fixing a different bug? Just curious.

Reply:
@bryphe potential overzealousness on my part I use [`shellcheck`](https://github.com/koalaman/shellcheck/wiki/SC2086) to lint shell scripts and an error it flagged on multiple lines was that those variable were not quoted which helps prevent word splitting and globbing the link is to a wiki page where the rationale for that is explained, I initially made that change because I thought that script wasn't working and was what had led to the file args not being passed to oni on open. I'm more than happy to take them out since its not really (as far as I can tell related to the mac os no file args issue)